transitmatters/t-performance-dash
# Slow Zones map visualization

This PR adds:

- A generalized `LineMap` component that uses a rewritten version of the NTT line rendering engine to draw strip maps
- A `SlowZonesMap` component that uses `LineMap` to show slow zones
- Storybook, for helping us debug/test React components.

There's still work I want to do on this: tooltips, layout edge cases, etc. But I think it's in a good state to share!

<img width="1904" alt="image" src="https://user-images.githubusercontent.com/2208769/229685915-33b63a3a-cc0f-4361-a19e-76df984414ab.png">

_Test plan:_
- Visit the Slow Zones tab for each of Red, Orange, Blue and make sure it renders
- Run `npm run storybook` and make sure that works too.
